gtk/src/subclass/cell_renderer_text.rs

```diff
@@ -38,11 +37,11 @@ impl<T: CellRendererTextImpl> CellRendererTextImplExt for T {
     }
 }
 
-unsafe impl<T: CellRendererTextImpl> IsSubclassable<T> for CellRendererTextClass {
-    fn override_vfuncs(&mut self) {
-        <CellRendererClass as IsSubclassable<T>>::override_vfuncs(self);
+unsafe impl<T: CellRendererTextImpl> IsSubclassable<T> for CellRendererText {
+    fn override_vfuncs(class: &mut ::glib::object::Class<Self>) {
+        <CellRenderer as IsSubclassable<T>>::override_vfuncs(class);
         unsafe {
-            let klass = &mut *(self as *mut Self as *mut gtk_sys::GtkCellRendererTextClass);
+            let klass = &mut *(class as *mut _ as *mut gtk_sys::GtkCellRendererTextClass);
```

Comment:
```suggestion
            let klass = &mut *(class.as_mut() as *mut _ as *mut gtk_sys::GtkCellRendererTextClass);
```

Might be nicer but doesn't really matter

Reply:
Especially no wildcard cast anymore :)

Reply:
Oh I like it!

Reply:
`sed -E -i 's;klass as \*mut _ ;klass.as_mut() ;' */src/subclass/*.rs`

Changes this automatically btw

Reply:
Updated the commit so once the CI is through we can merge it here.

Reply:
Actually, that should be equivalent to just `let klass = class.as_mut();`. Which is exactly what I was thinking with the `AsMut` implementation.

https://github.com/gtk-rs/gtk-rs/pull/16

Reply:
Oh even simpler. Thanks!